GoogleSignIn/Tests/Unit/GIDGoogleUserTest.m

```diff
@@ -125,11 +124,23 @@ - (void)testUpdateAuthState {
   XCTAssertEqualObjects(user.idToken.tokenString, updatedIDToken);
   XCTAssertEqualWithAccuracy([user.idToken.expirationDate timeIntervalSince1970],
                              updatedIDTokenExpireTime, kTimeAccuracy);
+  XCTAssertEqualObjects(user.refreshToken.tokenString, kNewRefreshToken);
```

Comment:
We should test that GIDToken instances are unchanged in the scenario in which the token strings are unchanged.

Reply:
Added a new test case `testUpdateAuthStateWithUnchangedRefreshTokenAndIDToken`